config/yunikorn-history-server/config.yml

```diff
@@ -21,3 +21,14 @@ yhs:
 log:
   level: "INFO"
   json_format: false
+
+cors:
```

Comment:
As per @nikola-jokic's comment, this should live under the `yhs` block.

Reply:
Updated. 
I think we need to change the implementation of `processEnvVar`.  We support configuration through both the config files and ENV vars.  According to the current implementation, there is no way to define nested env variables of **deeper than a single level** using ENV vars.

For example, if we try to define the following cors configuration:

`YHS_YHS_CORS_ALLOWED_ORIGIN` `(prefix_nest1_next2_iam_nest_3)` will be converted to `yhs.cors_allowed_origin` but from the configuration file it is read as `yhs.cors.allowed_origin`

---
**Relevant code:**
https://github.com/sudiptob2/yunikorn-history-server/blob/c912b75a837050853da9917306dc5509636658ce/internal/config/config.go#L195-L211
```go
	if err := k.Load(env.Provider("YHS_", ".", processEnvVar), nil); err != nil {
		return nil, fmt.Errorf("error loading environment variables: %v", err)
	}


	return k, nil
}


// Removes the prefix "YHS_" and replaces the first "_" with "."
// YHS_PARENT1_CHILD1_NAME will be converted into "parent1.child1_name"
func processEnvVar(s string) string {
	s = strings.TrimPrefix(s, "YHS_")
	firstIndex := strings.Index(s, "_")
	if firstIndex > -1 {
		s = s[:firstIndex] + "." + s[firstIndex+1:]
	}
	return strings.ToLower(s)
}
```